db-engine/src/main/java/com/splicemachine/db/impl/sql/execute/TriggerEventActivator.java

```diff
@@ -206,22 +212,19 @@ private void initTriggerExecContext(Vector<AutoincrementCounter> aiCounters) thr
      */
     void reopen() throws StandardException {
         initTriggerExecContext(null);
-        setupExecutors(triggerInfo);
+        setupExecutors(triggerInfo, activation);
     }
 
-    private void setupExecutors(TriggerInfo triggerInfo) throws StandardException {
+    private void setupExecutors(TriggerInfo triggerInfo, Activation activation) throws StandardException {
         // The SET statement cannot be executed in parallel.
         String regex    =   "(^set[\\s]+)|([\\s]+set[\\s]+)";
         Pattern pattern =   Pattern.compile(regex, Pattern.CASE_INSENSITIVE);
 
         for (TriggerDescriptor td : triggerInfo.getTriggerDescriptors()) {
             TriggerEvent event = td.getTriggerEvent();
             if (td.isRowTrigger()) {
-                // disable concurrent trigger for spark execution due to the high overhead
-                // to create lcc and TriggerExecutionContext for each row and each trigger,
-                // each make non-cached dictionary calls for
-                // defaultroles, defaultSchema, storedPreparedStatement ...
-                boolean runConcurrently = !fromSparkExecution;
+                // Now both control and Spark can use concurrent row triggers.
+                boolean runConcurrently = true;
```

Comment:
Can you simplify the logic here by removing the dead code?

Reply:
Done